### ISSUE TITLE
Parent process not being killed when executing through a init script?

### ISSUE THREAD

**Issue by xRiot** [toddbluhm/env-cmd#104]
**Overview**
We discovered an issue with running env-cmd through a init script, the parent is not being killed when we use `CTRL+C` to terminate the node express process from the terminal.

**Error message:**
```
Unable to terminate parent process successfully
```

**Background**
We basically have a crontab job that starts our node server on boot with a `tmux` session, this way we can easily restart it or view the console and _this approach is just a proof of concept, we'll be using a process manager such as PM2 down the road for production_. 

**Setup**
crontab -e _(this spawns a init session, there's no user and it's the system running this)_:
```
@reboot tmux new -d -s node && tmux send-keys -t node.0 "yarn --cwd /home/node start:staging" ENTER
```
package.json _(scripts section)_:
```
"start:staging": "env-cmd -e staging --use-shell 'wait-on tcp:27017 && node server.js --dev'"
```

**Dependencies:**
```
Distro: Ubuntu 18.04.3 LTS
Node: v8.10.0
Express: v4.17.1
env-cmd: v10.0.1
```

The **above** approach does **not** generate a stacktrace, we played around with the env-cmd command and we managed to generate one this way:
```
root@ip-172-31-40-55:/home/node# yarn env-cmd -e staging 'node server.js --dev'
yarn run v1.19.1
$ /home/node/node_modules/.bin/env-cmd -e staging 'node server.js --dev'
spawn node server.js --dev ENOENT
internal/process.js:178
        throw new Error(`Unknown signal: ${sig}`);
        ^

Error: Unknown signal: 1
    at process.kill (internal/process.js:178:15)
    at TermSignals._terminateProcess (/home/node/node_modules/env-cmd/dist/signal-termination.js:47:28)
    at process.terminateSpawnedProcessFuncHandlers.(anonymous function) (/home/node/node_modules/env-cmd/dist/signal-termination.js:20:30)
    at Object.onceWrapper (events.js:315:30)
    at emitOne (events.js:116:13)
    at process.emit (events.js:211:7)
    at process.exit (internal/process.js:157:15)
    at TermSignals._uncaughtExceptionHandler (/home/node/node_modules/env-cmd/dist/signal-termination.js:68:17)
    at process.on (/home/node/node_modules/env-cmd/dist/signal-termination.js:40:53)
    at emitOne (events.js:116:13)
error Command failed with exit code 7.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Let us know if you need more information, thanks!

**Comment by toddbluhm** [comment]
Yeah, this is very helpful. It looks like `sig` is coming in as a number rather than a signal string. I can add a safety check if `sig` is a non-zero number then act appropriately.

**Comment by toddbluhm** [comment]
This should now be resolved by commit: https://github.com/toddbluhm/env-cmd/commit/f92f8b51ff642448b4cd6d855ad88e4e28058216

Feel free to let me know if it is working or close the ticket if it is not needed anymore. I will update this thread once the release is out.

**Comment by toddbluhm** [comment]
I have released `10.1.0` with the fix for this issue.
